daily.js

```diff
@@ -1,9 +1,8 @@
 const { DateTime } = require('luxon')
 
-const {
-  sources: { github: { issuesByRange, reposByRange, enrichIssues, ignoreProjects, ignoreBotUsers, formatPreviously } },
-  targets: { slack: { uploadMD } },
-} = require('.')
+const { issuesByRange, reposByRange, enrichIssues, ignoreProjects, ignoreBotUsers, formatPreviously } = require('./sources/github')
+const { getVacays, formatVacays } = require('./sources/asana')
```

Comment:
why adding these files to `index.js` if fetching from their original `dir`?

Reply:
wanted to move them in backward compatible ways, with a remote chance of other ppl already using this as a library and used to importing things from root `.`.